### PR TITLE
refactor(code): Codename: Happy Linter

### PR DESCRIPTION
**Pull request by danielsogl**
I removed some whitespaces and refactored the code.
I didn't change any line of code

### CODE REVIEW

**Comment by dwieeb** [comment]
@danielsogl What linter/style guide are you using?

**Comment by danielsogl** [comment]
You find it here: https://github.com/vvakame/typescript-formatter/blob/master/tsconfig.json

And actually I also use atom-beautify. The plugin also uses this tsconfig file

File: packages/cli-plugin-cordova/src/commands/resources.ts [hunk 1]
```diff
@@ -133,149 +133,149 @@ export class ResourcesCommand extends Command implements CommandPreRun {
       .filter((img) => resourceTypes.includes(img.resType))
       .map((img) => ({
         ...img,
-        dest: path.join(resourceDir, img.platform, img.resType, img.name)
```
Comment (dwieeb) [inline packages/cli-plugin-cordova/src/commands/resources.ts]: ??

File: packages/cli-plugin-cordova/src/commands/resources.ts [hunk 2]
```diff
@@ -133,149 +133,149 @@ export class ResourcesCommand extends Command implements CommandPreRun {
...
+      dest: path.join(resourceDir, img.platform, img.resType, img.name)
       }));
 
-    if (platform) {
```
Comment (dwieeb) [inline packages/cli-plugin-cordova/src/commands/resources.ts]: wrong

File: packages/cli-plugin-cordova/src/commands/resources.ts [hunk 3]
```diff
@@ -133,149 +133,149 @@ export class ResourcesCommand extends Command implements CommandPreRun {
...
+  if (platform) {
+    imgResources = imgResources.filter((img) => img.platform === platform);
+  }
 
     this.env.log.debug(`imgResources=${imgResources.length}`);
 
-    // Create the resource directories that are needed for the images we will create
-    const buildDirResponses = await createImgDestinationDirectories(imgResources);
-    this.env.log.debug(`${chalk.green('createImgDestinationDirectories')} completed - length=${buildDirResponses.length}`);
+// Create the resource directories that are needed for the images we will create
```
Comment (dwieeb) [inline packages/cli-plugin-cordova/src/commands/resources.ts]: root level now?

File: packages/cli-plugin-cordova/src/lib/utils/configXmlUtils.ts [hunk 4]
```diff
@@ -159,7 +159,7 @@ export async function writeConfigXmlContentSrc(projectDir: string, devServerUrl:
   // If there is no content element then throw an error
   if (!configJson.widget.content) {
     throw new Error('\nYour config.xml file does not have a <content> element. ' +
-                    '\nAdd something like: <content src="index.html"/>');
+      '\nAdd something like: <content src="index.html"/>');
```
Comment (dwieeb) [inline packages/cli-plugin-cordova/src/lib/utils/configXmlUtils.ts]: seems uglier to me

File: packages/cli-utils/src/lib/http.ts [hunk 5]
```diff
@@ -48,29 +48,28 @@ export function createRequest(method: string, url: string): superagentType.Super
...
-                             + formatAPIError(req, r));
+        + formatAPIError(req, r));
     }
 
     return r;
   }
 
   paginate<T extends Response<Object[]>>(reqgen: () => superagentType.SuperAgentRequest, guard: (res: APIResponseSuccess) => res is T): Paginator<T> {
-    return new Paginator<T>(this, reqgen, guard);
+    return new Paginator<T> (this, reqgen, guard);
```
Comment (dwieeb) [inline packages/cli-utils/src/lib/http.ts]: no...

File: packages/cli-utils/src/lib/login.ts [hunk 6]
```diff
@@ -3,18 +3,18 @@ import { validators } from './validators';
...
     type: 'input',
     name: 'email',
     message: 'Email:',
     validate: (input: string) => validators.email(input),
   }, {
-    type: 'password',
-    name: 'password',
-    message: 'Password:'
-  }]);
+      type: 'password',
```
Comment (dwieeb) [inline packages/cli-utils/src/lib/login.ts]: ???

File: packages/cli-utils/src/lib/package.ts [hunk 7]
```diff
@@ -42,117 +42,118 @@ export class PackageClient {
...
 
-    return res.data;
+return res.data;
   }
 
-  async requestProjectUpload(): Promise<PackageProjectRequest> {
-    const req = this.client.make('POST', '/package/projects')
-      .set('Authorization', `Bearer ${this.appUserToken}`)
-      .send({});
+async requestProjectUpload(): Promise < PackageProjectRequest > {
```
Comment (dwieeb) [inline packages/cli-utils/src/lib/package.ts]: what? does this even compile?

File: packages/cli-utils/src/lib/package.ts [hunk 8]
```diff
@@ -42,117 +42,118 @@ export class PackageClient {
...
+    .set('Authorization', `Bearer ${this.appUserToken}`)
+    .send({});
 
-    const res = await this.client.do(req);
+  const res = await this.client.do(req);
 
-    if (!isPackageProjectRequestResponse(res)) {
-      throw createFatalAPIFormat(req, res);
-    }
+  if (!isPackageProjectRequestResponse(res)) {
```
Comment (dwieeb) [inline packages/cli-utils/src/lib/package.ts]: this isn't indented properly

**Comment by danielsogl** [comment]
Looks like the linter fails this time lol

**Comment by dwieeb** [comment]
You linked me to a tsconfig file--what does it have to do with linting?

Whatever formatter you used kinda had disastrous changes. Did you look at the code output? I looked through just a couple files and found so many things wrong.

We're using tslint at the moment with minimal options (`npm run lint`). I would prefer to expand upon the tooling we already have than use a formatter in a text editor (I don't use atom)

**Pull request closed without merging**